frontend/src/components/Per-ImplementationPage/DialectCompliance.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Card, Table } from "react-bootstrap";
+import { ReportData } from "../../data/parseReportData";
+
+interface DialectComplianceProps {
+  loaderData: ReportData;
```

Comment:
This seems to not be of `ReportData` type? it causesa bunch of TS errors below

Reply:
Fixed